### PR TITLE
fix busted CI

### PR DESCRIPTION
**Pull request by ryanpetrello**

### CODE REVIEW

File: requirements/requirements_dev.txt [hunk 1]
```diff
@@ -13,7 +13,7 @@ pytest-django
 pytest-pythonpath
 pytest-mock
 pytest-timeout
-pytest-xdist
+pytest-xdist<1.28.0
```
Comment (ryanpetrello) [inline requirements/requirements_dev.txt:16]: The latest xdist pins to pytest>=4 and we're not ready.

**Comment by ryanpetrello** [comment]
replaces https://github.com/ansible/awx/pull/3616

File: requirements/requirements_dev.txt (hunk 1 above)
Comment (ryanpetrello) [reply requirements/requirements_dev.txt:16]: cc @wenottingham

**Review by shanemcd — APPROVED** [review]

File: tools/docker-compose/Dockerfile [hunk 2]
```diff
@@ -19,9 +19,7 @@ RUN yum -y update && yum -y install openssh-server ansible mg vim tmux \
   zanata-python-client gettext gcc-c++ libcurl-devel bzip2 \
   rsync
 
-RUN ln -s /usr/bin/python36 /usr/bin/python3
```
Comment (ryanpetrello) [inline tools/docker-compose/Dockerfile:22]: Looks like there's a new centos7 image as of ~20 days ago that does this now.  We had an old cached one on our zuul nodes.

**Comment by ryanpetrello** [comment]
regate

**Comment by ryanpetrello** [comment]
regate

**Pull request merged**